base_layer/wallet/src/output_manager_service/mod.rs

```diff
@@ -44,10 +44,15 @@ const LOG_TARGET: &'static str = "wallet::output_manager_service::initializer";
 
 pub type TxId = u64;
 
+#[derive(Clone)]
+pub enum MasterKeyType {
+    MasterSeed(PrivateKey),
```

Comment:
I feel like this should always be a PrivateKey. Seeds words are simply a representation of that key, so `from_seed_words` and `to_seed_words` methods would allow to easily translate from the PrivateKey source of truth

Reply:
Fair enough, will remove for now and after we decide on seed backup methods (not for MVP as far as I can tell) we can implement the translation outside of the service as you suggest.